doc/news/changes/minor/20170623DanielArndt

```diff
@@ -0,0 +1,5 @@
+New: The class Timer has the new functions get_total_data(),
+print_total_data, last_wall_time, cpu_time and last_cpu_time()
+for accessing information about the total run and the last lap now.
```

Comment:
There are just too many Germanisms in this sentence for me to let it go :-)

Write this as "The Timer class now has new functions Timer::get_total_data(), Timer::print_total_data(), ... to access information about the total run time as well as the last lap."